Frontend/src/components/TableEditors/Items/ItemPopupFormular.svelte

```diff
@@ -138,17 +157,18 @@
 
             {
                 id: "status_on_website",
+                disabled: docIsDeleted,
                 label: "Status auf Webseite",
                 group: "Status",
                 type: InputTypes.SELECTION,
                 bindTo: {
-                    keyValueStoreKey: "currentDoc",
+                    keyValueStoreKey: docIsDeleted ? "mock" : "currentDoc",
                     attr: "status_on_website",
                 },
                 selectionOptions: [
-                    { value: "deleted", label: "gelöscht" },
                     { value: "instock", label: "verfügbar" },
                     { value: "outofstock", label: "verliehen" },
+                    { value: "onbackorder", label: "nicht verleihbar" },
```

Comment:
"onbackorder" is the right status to use here correct? @skjerns

Reply:
Yes, I named it "aus Sortiment genommen"

Reply:
I needed something shorter so it looks good :)